apps/prairielearn/assets/scripts/studentAssessmentInstanceClient.ts

```diff
@@ -0,0 +1,9 @@
+$(function () {
+  $('[data-toggle="popover"]').popover({ sanitize: false, container: 'body' });
+});
+
+// make the file inputs display the file name
+$(document).on('change', '.custom-file-input', function () {
+  const filename = $(this).val().replace(/\\/g, '/').replace(/.*\//, '');
+  $(this).parent('.custom-file').find('.custom-file-label').text(filename);
+});
```

Comment:
I'm seeing both of these constructs repeated in multiple files. Do you think it's worth moving them to asset libs and importing them here? Or at least the custom-file-input handler, which is also used in questions?

Reply:
Yeah, I think it's reasonable to do that for the custom file input. Will change.

Reply:
Done in 50791f3.